app/controllers/serials_controller.rb

```diff
@@ -1,6 +1,8 @@
 class SerialsController < ApplicationController
   def index
     @serials = current_site.serials.published.order(id: :desc).page(params[:page])
```

Comment:
We discussed on chat about this topic.
We decided name this order specification as scope and use it here.
* To specify order for pagination.
* To be meaningful for programmer.